src/app/blocks/actions-block/actions-block.component.html

```diff
@@ -1,7 +1,7 @@
 <ng-container *ngFor="let b of buttons; let i = index">
-  <button type="button" class="actions-button" *ngIf="!b.color || b.color !== 'link'"
+  <button [attr.id]="b.id" type="button" class="actions-button" *ngIf="b.enabled && (!b.color || b.color !== 'link')"
           mat-flat-button [color]="b.color" (click)="onSubmit(b)">{{ getLabel(b) }}</button>
-  <a class="actions-button" *ngIf="b.color && b.color === 'link'"
+  <a [attr.id]="b.id" class="actions-button" *ngIf="b.color && b.color === 'link'"
```

Comment:
Should there have been an additional e.enabled check here?
Seperately, I wonder what's going on with this color stuff.

Reply:
Yeah, missing the second check. 

I've removed one of the checks for b.color. It would result in the button not being displayed if no colour was set. Now that I've added a direct way to disable a button, this should be removed.